analytics_data_api/v0/views/learners.py

```diff
@@ -85,3 +82,136 @@ def get_object(self, queryset=None):
         if len(queryset) == 1:
             return queryset[0]
         raise LearnerNotFoundError(username=self.username, course_id=self.course_id)
+
+
+class LearnerListView(CourseViewMixin, generics.ListAPIView):
+    """
+    Get a paginated list of student data for a particular course.
+
+    **Example Request**
+
+        GET /api/v0/learners/?course_id={course_id}
+
+    **Response Values**
+
+        Returns a paginated list of learner metadata and engagement data.
+        Pagination data is returned in the top-level of the returned JSON
+        object:
+
+            * count: The number of learners matching the query
+            * page: The current one-indexed page number
+            * next: A hyperlink to the next page if one exists, otherwise null
+            * previous: A hyperlink to the previous page if one exists, otherwise null
+
+        The 'results' key in the returned JSON object maps to an array of
+        learners which contains at most a full page's worth of learners.  Each
+        learner is a JSON object containing the following keys:
+
+            * username: User name.
+            * enrollment_mode: Enrollment mode (e.g. "honor).
+            * name: User name.
```

Comment:
`username` and `name` have the same description.  Is name is the last and first name?